crates/forge/src/runner.rs

```diff
@@ -560,10 +573,50 @@ impl<'a> ContractRunner<'a> {
                 traces,
                 labeled_addresses,
                 kind: TestKind::Standard(0),
+                debug,
+                breakpoints,
                 ..Default::default()
             }
         }
 
+        // if should debug
+        if self.debug {
+            let mut debug_executor = self.executor.clone();
+            // turn the debug traces on
+            debug_executor.inspector.enable_debugger(true);
+            debug_executor.inspector.tracing(true);
+            let calldata = if let Some(counterexample) = result.counterexample.as_ref() {
+                match counterexample {
+                    CounterExample::Single(ce) => ce.calldata.clone(),
+                    _ => unimplemented!(),
```

Comment:
will we resolve this?

Reply:
I don't know, I'm not sure how to handle this because everything has been shaped for one `calldata` and here it messes things up to have a sequence of calls

Reply:
Could we do that in #3 ? Probably modify the `single_fuzz` to take in a `Vec<calldata, value>` to support sequence calls. Just noticed that the value is defaulted to 0 right now as well, might make it a bug

Reply:
I'm just not familiar how we can force the fuzzer to construct a sequence of calls